tracer/src/Datadog.Trace/Configuration/ImmutableTracerSettings.cs

```diff
@@ -267,6 +268,12 @@ public ImmutableTracerSettings(TracerSettings settings)
         /// <seealso cref="ConfigurationKeys.FeatureFlags.RouteTemplateResourceNamesEnabled"/>
         internal bool RouteTemplateResourceNamesEnabled { get; }
 
+        /// <summary>
+        /// Gets a value indicating whether to enable the updated WCF instrumentation that delays execution
+        /// until later in the WCF pipeline when the WCF server exception handling is established.
+        /// </summary>
+        internal bool DelayWcfInstrumentationEnabled { get; }
```

Comment:
Grammatically, either `WcfInstrumentationDelayEnabled` or `DelayedWcfInstrumentationEnabled` sounds better to me. If this is only meant as a temporary feature flag, then my level of care is lower.

Reply:
We have one customer using this already (via environment variable `DD_TRACE_DELAY_WCF_INSTRUMENTATION_ENABLED`) so I'd rather not change it. Fingers crossed we can make this the default behavior once we figure out some missing spans